apps/src/StudioApp.js

```diff
@@ -2850,9 +2850,13 @@ StudioApp.prototype.handleUsingBlockly_ = function (config) {
     );
 
     const toolboxWithoutWhitespace = config.level.toolbox.replace(/\s/g, '');
+    // An empty toolbox should be treated as no toolbox.
     if (
       toolboxWithoutWhitespace === '<xml></xml>' ||
-      toolboxWithoutWhitespace === '<xml/>'
+      toolboxWithoutWhitespace === '<xml/>' ||
+      // Google Blockly always adds the xmlns attribute.
+      toolboxWithoutWhitespace ===
+        '<xmlxmlns="https://developers.google.com/blockly/xml"/>'
```

Comment:
Is there a missing space between xml and xmlns?

Reply:
There is because `toolboxWithoutWhitespace` strips all whitespace from the string (probably to avoid diffferences in spacing around `\>`)

Reply:
Got it. And it doesn't look like we use `toolboxWithoutWhitespace` anywhere else, so that's good. Is it possible Google Blockly would ever give separate open/close tags like `<xml xmlns="..."></xml>`? I'm guessing we haven't seen it.

Reply:
They don't currently, but that doesn't mean they won't. I can add it as an option